data/json/items/armor.json

```diff
@@ -5809,6 +5809,79 @@
     "material_thickness": 1,
     "flags": [ "WAIST", "OVERSIZE", "WATER_FRIENDLY" ]
   },
+  {
+    "id": "ammo_pouch",
+    "type": "ARMOR",
+    "name": "ammo pouch",
+    "name_plural": "ammo pouches",
+    "description": "A small pouch that can be used to store most types of small ammunition, rockets will not fit.  Activate to store ammunition.",
+    "looks_like": "leather_pouch",
+    "weight": 490,
+    "volume": 2,
+    "price": 1000,
+    "to_hit": 1,
+    "bashing": 1,
+    "material": [ "leather" ],
+    "symbol": "[",
+    "color": "brown",
+    "covers": [ "TORSO" ],
+    "coverage": 10,
+    "encumbrance": 2,
+    "material_thickness": 1,
+    "use_action": {
+      "type": "bandolier",
+      "capacity": 60,
+      "ammo": [
+        "pebble",
+        "700nx",
+        "50",
+        "4570",
+        "BB",
+        "410shot",
+        "22",
+        "shot",
+        "9mm",
+        "357sig",
+        "9x18",
+        "380",
+        "38",
+        "40",
+        "10mm",
+        "44",
+        "45",
+        "460",
+        "454",
+        "45colt",
+        "500",
+        "57",
+        "46",
+        "762",
+        "545x39",
+        "223",
+        "3006",
+        "270win",
+        "308",
+        "12mm",
+        "8x40mm",
+        "20x66mm",
+        "5x50",
+        "metal_rail",
+        "stimpack_ammo",
+        "ampoule",
+        "300",
+        "32",
+        "762x25",
+        "flintlock",
+        "36paper",
+        "44paper",
+        "762R",
+        "paintball",
+        "crystallized_mana"
```

Comment:
Don't use item from mods (e.g. `crystallzied_mana` from `Magiclysm`) - it will cause errors when mods are not used.

Reply:
ok thank you, do you know how I could add it through the mod? I am not familiar enough with the code for mods